### PR TITLE
Fix when the data is the same as stored

### PR DESCRIPTION
**Pull request by fredericgboutin-yapla**
From https://github.com/Shardj/zf1-future/commit/9175301058d324bcdcdca0d091b2e6c394273a95

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "zf1s/zf1",
+    "name": "yapla/zf1",
```
Comment (fredericgboutin-yapla) [inline composer.json:2]: ok. Obviously I don't want to merge this here but I needed this definition for `composer require` my forked version :think:

**Comment by glensc** [comment]
ideally:
1. the changes should be squashed to single commit by contributor
1. as it's contribution by other person, it should be attributed: `Co-Authored-By: Original Author <their@email.xx>` in git trailer, or git commit --amend "Original Author <their@email.xx>"  and `Co-Authored-By` the second author who made this commit
2. the original author, the original pull request should be mentioned in pull request body.
3. never create pull requests from your `master` branch.

but I guess it's ok as well if maintainers just use github squash merge option and fill the `Co-Authored-By` as git trailer manually in commit message.

**Comment by glensc** [comment]
@fredericgboutin-yapla you could also remove the two unwanted commits from your master and force push

```
git reset --hard eb369bf843c3c01385adde1c246daac7eccc2977
git push --force-with-lease
```

**Comment by fredericgboutin-yapla** [comment]
I'm grateful for your remarks. TBH this is literally the first time I'm contributing on a project like this. Is there anything I should've read first?

**Comment by fredericgboutin-yapla** [comment]
I started from scratch using Github Desktop - https://github.com/zf1s/zf1/pull/164

**Comment by glensc** [comment]
cross-link the PR's as well, saying in new pr body: `Replaces <link to the old pr>`

also worth to tag the username of original commit there as well: `Originally submitted to zf1-future by @githubaccount`

**Comment by fredericgboutin-yapla** [comment]
I'm struggling to find any original pull request or PR 😓

**Comment by glensc** [comment]
if you click on the commit it's the `master (#23)`

image

**Pull request closed without merging**